최창익/app/src/main/java/com/mash_up/mvvmstudy/repository/model/Repositories.kt

```diff
@@ -0,0 +1,62 @@
+package com.mash_up.mvvmstudy.repository.model
+
+import com.google.gson.annotations.SerializedName
+import java.io.Serializable
+
+data class Repositories(
+    @SerializedName("total_count")
+    val totalCount: Int,
+    @SerializedName("incomplete_results")
+    val incompleteResult: Boolean,
+    @SerializedName("items")
+    val repositories: List<Repository>
+)
+
+data class Repository(
+    @SerializedName("id")
+    val id: Long,
+    @SerializedName("name")
+    val name: String,
+    @SerializedName("full_name")
+    val fullName: String,
+    @SerializedName("owner")
+    val owner: Owner,
+    @SerializedName("language")
+    val language: String,
+    @SerializedName("stargazers_count")
+    val stargazersCount: Int,
+    @SerializedName("description")
+    val description: String,
+    @SerializedName("updated_at")
+    val updatedAt: String,
```

Comment:
description과 language가 null 인 Item이 있어서 nullable하게 선언해줘야 할 것 같아요!
```suggestion
    @SerializedName("id")
    val id: Long,
    @SerializedName("name")
    val name: String,
    @SerializedName("full_name")
    val fullName: String,
    @SerializedName("owner")
    val owner: Owner,
    @SerializedName("language")
    val language: String?,
    @SerializedName("stargazers_count")
    val stargazersCount: Int,
    @SerializedName("description")
    val description: String?,
    @SerializedName("updated_at")
    val updatedAt: String,
```

Reply:
null인 데이터를 받으면 크래시가 날 뻔 했군요...감사합니다